kmath-core/src/commonMain/kotlin/space/kscience/kmath/operations/Algebra.kt

```diff
@@ -114,15 +115,6 @@ public interface SpaceOperations<T> : Algebra<T> {
      */
     public fun add(a: T, b: T): T
 
-    /**
-     * Multiplication of element by scalar.
-     *
-     * @param a the multiplier.
-     * @param k the multiplicand.
-     * @return the produce.
-     */
-    public fun multiply(a: T, k: Number): T
```

Comment:
Since the scalar multiplication was removed, this interface must be called GroupOperations.